.github/workflows/community-releases.yml

```diff
@@ -71,28 +115,37 @@ jobs:
             --platform=linux/amd64,linux/arm64 --provenance=false
             --output=type=image,name=ghcr.io/${{ env.REGISTRY_OWNER }}/${{ env.SERVICE_NAME }},push-by-digest=true,name-canonical=true
         run: |
-          cd apps/$SERVICE_NAME && pnpm run docker:build
+          if [ "${{ env.SERVICE_NAME }}" = "embed" ] || [ "${{ env.SERVICE_NAME }}" = "embed-ee" ]; then
```

Comment:
@AliaksandrRyzhou can you please elaborate on this? Does embed differ between community and ee version? cc: @LetItRock

Reply:
I'm not 100% sure about it, I just decided so because we have a private package repo embed-ee. Maybe @LetItRock  can clarify this moment

Reply:
@LetItRock,  after all, should I delete `embed-ee` from the matrix because embed is not different from the community build? Why do we have the `embed-ee` package in the GitHub packages?